net/http/authz/grant.go

```diff
@@ -1,4 +1,75 @@
 package authz
 
+import (
+	"bytes"
+	"context"
+	"time"
+
+	"github.com/golang/protobuf/proto"
+
+	"chain/database/raft"
+	"chain/errors"
+)
+
 // Generate code for the Grant and GrantList types.
 //go:generate protoc -I. -I$CHAIN/.. --go_out=. grant.proto
+
+// StoreGrant stores a new grant in the provided raft store
+func StoreGrant(ctx context.Context, raftDB *raft.Service, grant Grant, grantPrefix string) error {
```

Comment:
Didn't scrutinize this too closely...is this just a copy/paste? If not I can review more thoroughly.